src/pages/HomePage/Header.jsx

```diff
@@ -27,13 +27,15 @@ const Header = ({ pricingRef, contactRef, featureRef, onButtonClick }) => {
             />
           </Hidden>
           <Hidden mdDown>
-            <div style={{ display: 'flex', alignItems: 'center' }}>
+            <div style={{ display: "flex", alignItems: "center" }}>
               <Tabs textColor="inherit">
                 <Tab label="Features" onClick={featureRef} />
                 <Tab label="Pricing" onClick={pricingRef} />
                 <Tab label="Contact" onClick={contactRef} />
               </Tabs>
-
+              <StyledButton style={{}} onClick={onButtonClick}>
+                Login
+              </StyledButton>
```

Comment:
Great! This mirrors the functionality of the signup button. In HomePage.jsx, let's create a new function that redirects the user to the signup page when they click the signup button. Then, we can prop-drill this function to the header component, similar to how onButtonClick is currently handled.

Reply:
function created